pkg/integrations/v4/logs/cfg.go

```diff
@@ -344,6 +383,87 @@ func parseTcpInput(l LogCfg) (input FBCfgInput, filters []FBCfgParser, err error
 	return input, filters, nil
 }
 
+//Winlog: "winlog" plugin
+func parseWinlogInput(l LogCfg, dbPath string) (input FBCfgInput, filters []FBCfgParser, err error) {
+	input = newWinlogInput(*l.Winlog, dbPath, l.Name)
+	filters = append(filters, newRecordModifierFilterForInput(l.Name, fbInputTypeWinlog, l.Attributes))
+	scriptContent, err := createLuaScript(*l.Winlog)
+	if err != nil {
+		return FBCfgInput{}, []FBCfgParser{}, err
+	}
+	scriptName, err := saveToTempFile([]byte(scriptContent))
+	if err != nil {
+		return FBCfgInput{}, []FBCfgParser{}, err
+	}
+	eventIdLuaFilter := newLuaFilter(l.Name, scriptName)
+	filters = append(filters, eventIdLuaFilter)
+
+	return input, filters, nil
+}
+
+func createLuaScript(winlog LogWinlogCfg) (scriptContent string, err error) {
+	var fbLuaScript FBLuaScript
+	fbLuaScript.FnName = fbLuaFnName
+	included, excluded := winlog.CollectEventIds, winlog.ExcludeEventIds
+	fbLuaScript.IncludedEventIds, err = createConditions(included, "true")
+	if err != nil {
+		return "", err
+	}
+	fbLuaScript.ExcludedEventIds, err = createConditions(excluded, "false")
+	if err != nil {
+		return "", err
+	}
+	scriptContent, err = fbLuaScript.Format()
+	if err != nil {
+		return "", err
+	}
+
+	return scriptContent, err
+}
+
+func createConditions(numberRanges []string, defaultIfEmpty string) (conditions string, err error) {
```

Comment:
I think we should add a test targeting this specific function, what do you think?

Reply:
I agree, I added it.